src/bacnet/basic/object/sc_netport.c

```diff
@@ -656,6 +714,15 @@ bool Network_Port_SC_Direct_Connect_Initiate_Enable_Set(
         return false;
     params->SC_Direct_Connect_Initiate_Enable = value;
 
+    return true;
```

Comment:
@Michail-Antropov This difference shows that Network_Port_Changes_Pending_Set() was dropped from the definition of Network_Port_SC_Primary_Hub_URI_Set(). Is that intentional?

Reply:
Yes, the flag Changes Pending is set Network_Port_Write_Property(). Otherwise, the flag will be set for each setter call, incl. during normal initialization the property.